frontend/wikiPages/ca/wiki/Welcome

```diff
@@ -0,0 +1,83 @@
+<h1>Benvingut</h1>
+
+<div class="content">
+
+<p>Heu arribat a la Pàgina Web de <b>PhiloBiblon </b>una base de dades biobibliogràfica sobre textos antics escrits en diverses llengües romàniques en la península Ibèrica en l'edat mitjana i inicis del Renaixement, informació accessible lliurement a través d'Internet.</p>
+
+<p>Una cerca a <a href="ca/search/texid/query">Obra</a> ens proporciona una llista de registres que contenen els termes inclosos a la cerca. F
```

Comment:
We need to modify all the search links by just adding a trailing slash.
For example in this case:
```
<a href="ca/search/texid/query">Obra</a>
```
to
```
<a href="ca/search/texid/query/">Obra</a>
```